public/typography.css

```diff
@@ -144,3 +144,8 @@ p,
   text-transform: uppercase;
   letter-spacing: 0.05rem;
 }
+
+/* Disables contextual alternates */
+.typo-disable-calt {
```

Comment:
In which circumstances would we want this contextual alternate representation?
Should we not just disable it globally?

We set other font flags here: https://github.com/radicle-dev/radicle-upstream/blob/master/public/global.css#L13.

Reply:
Yeah I wish we could disable it globally only for certain characters, but that doesn't seem to be possible. Because the contextual alternates in Inter are really cool, I'd like to keep them around and just target certain strings where we specifically find a problem.

But… maybe we should just disable them globally since we have font-choosing now as a setting, and this style only applies to the Inter font.

🤔

Reply:
I think this implementation makes sense. Cause Inter is still the default.